code/game/objects/structures/watercloset.dm

```diff
@@ -478,3 +478,45 @@
 	icon_state = "puddle-splash"
 	. = ..()
 	icon_state = "puddle"
+
+
+//Shower Curtains//
+
+#define SHOWER_OPEN_LAYER OBJ_LAYER + 0.4
+#define SHOWER_CLOSED_LAYER MOB_LAYER + 0.2
```

Comment:
layer defines should be kept in layers.dm
also if possible reuse layer defines instead of adding new ones